README.md

```diff
@@ -1,2 +1,53 @@
-# atlas
-SPA that provides easy access to basic information for all countries.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Atlas
+
+SPA application that provides users easy access to information regarding all countries on the planet. Built with React, Redux, Tailwind and testing with Jest.
+## Built With
+
+- Javascript
+- React
+- Redux
+- Tailwind
+
+## Getting Started
+To get a local copy up and running follow these simple steps.
+### Prerequisites
+- Having a modern internet browser: I.E. Chrome, Edge, Safari, Firefox.
+### Setup
+- Go into the main page for this project: https://github.com/lu-jim/atlas
+- Click the green Code button next to the About section.
+- Select the then click the 📋 button.
+### Install
+- Open your console and change the current working directory into the destination for this project.
+- Type `git clone`, and then paste the URL copied in the Setup section.
+- Type Enter to finish the installation process.
+- Run `npm i` and after all the dependencies are installed run `npm start`
+
```

Comment:
- [ ] [OPTIONAL]Kindly add a command to run the tests in your README file to help other people to be able to know the command to run the tests, you may add something like this: run `npm test` to run the tests

Reply:
Done!